cypress/integration/plugins/alerting-dashboards-plugin/monitors_dashboard_spec.js

```diff
@@ -38,7 +38,7 @@ const clusterHealthMonitor = {
         severity: '1',
         condition: {
           script: {
-            source: 'ctx.results[0].status != "green"',
+            source: 'ctx.results[0].status != "blue"',
```

Comment:
For context, this was changed to `blue` because the test was failing when run against multinode clusters. The indexes used by the tests were only reliably in a `yellow` state on single node clusters because they were configured to have replicas. Rather than reconfigure the test indexes, it was less invasive for other tests to adjust the condition used by this test.

Reply:
Could we add comments of this change with the context,so we can understand in future when review the code